preprocessing/DataLoader.py

```diff
@@ -131,4 +131,4 @@ def runDataLoader():
 
 def runQuickLoader():
     all_json, metadata = getAllData()
-    return getDataFrame(all_json[:1000], metadata)
+    return getDataFrame(all_json[:100], metadata)
```

Comment:
This is fine, but if you make changes just for your own testing, try not to push those changes.